### ISSUE TITLE
PC Shuts Down, Kernet-Power Error 41

### ISSUE THREAD

**Issue by RhanCandia** [IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT#211]
### Checklist [README]

- [X] Device is not undervolted nor overclocked
- [X] Device is using the [latest drivers](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/wiki/Intel-driver-updates-and-changelogs)
- [X] Game is not cracked, modded and use the latest patch

### Game / Application [Required]

Control

### Game Platform [Required]

- [ ] Steam
- [ ] Microsoft Store
- [ ] GOG Galaxy
- [ ] Ubisoft Connect
- [ ] Battle.net
- [ ] Epic Games
- [ ] Rockstar Games Launcher
- [X] EA app
- [X] Other (please specify)

### Other game platform

Xbox Game Pass PC

### Processor / Processor Number [Required]

Ryzen 5 5600

### Graphic Card [Required]

ASRock Arc A750

### GPU Driver Version [Required]

- [X] 31.0.101.3975 (*latest*)
- [X] 31.0.101.3959
- [ ] 31.0.101.3802
- [ ] 31.0.101.3793
- [ ] 31.0.101.3491
- [ ] 31.0.101.3490
- [ ] 31.0.101.3430
- [ ] 31.0.101.3276
- [ ] 30.0.101.3268
- [ ] 30.0.101.3259
- [ ] 30.0.101.1743

### Rendering API [Required]

- [ ] Vulkan
- [ ] OpenGL
- [X] DirectX12
- [ ] DirectX11
- [ ] DirectX10
- [ ] DirectX9

### Windows Build Number [Required]

- [X] Windows 11 22H2
- [ ] Windows 11 21H1
- [ ] Windows 10 22H2
- [ ] Windows 10 21H2
- [ ] Windows 10 21H1
- [ ] Other (Please specify)

### Intel System Support Utility report

[SSU.txt](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/files/10323362/SSU.txt)

### Description and steps to reproduce [Required]

Run CONTROL
Go to Graphics Settings
Max out graphical settings
Enable Ray Tracing set to High

### Game graphic quality [Required]

- [ ] Low
- [ ] Medium
- [X] High
- [X] Ultra

### Game resolution [Required]

1920x1080

### Game VSync [Required]

On

### Game display mode [Required]

- [X] Fullscreen
- [ ] Windowed
- [ ] Borderless Windowed

### Detailed game settings [Required]

I just use the highest preset and then I enable Ray Tracing set to High or enable all ray tracing options.

**Comment by RhanCandia** [comment]
For this one I am suspecting that my PSU is failung to supply the power needed because I am running this with a 450W PSU.

I already ordered a 650W Gold rated PSU and hopefully it arrives next week. If that solves the issue, I will close this ticket. If I still encounter it even with a better PSU then I will post updates here on this issue.

Is this okay? Thanks!

**Comment by IntelSupport-Arun** [comment]
Hi @RhanCandia ,

Thank you for contacting us.
Sure, you may check with the 650W Gold rated PSU and see the issue is resolved or not.
I will wait for your update.

**Comment by IntelSupport-Arun** [comment]
Hi @RhanCandia ,

Did you receive the  650W PSU and tested it?

**Comment by RhanCandia** [comment]
> Hi @RhanCandia ,
> 
> Did you receive the 650W PSU and tested it?

I'm still waiting for it. I will give you an update when I get it. Thanks for following up.

**Comment by RhanCandia** [comment]
> Hi @RhanCandia ,
> 
> Did you receive the 650W PSU and tested it?

The higher capacity power supply arrived today and I already installed it. I have a 100% replication rate of this issue on my previous power 450w PSU.

On the Corsair VS450
1. Launch Suporposition Benchmark
2. Run 1080p Extreme or 4K Optimized
3. Instant shutdown
4. Kernel-Power Error Code 41

Now on my new BeQuiet Pure Power 11 FM 650W it does not shutdown anymore. I even went as far as assigning higher power limit through the Arc Control software. From 180 watts I pushed it to 200 watts. Running the same benchmarks above is still stable. No crash. No shutdowns.

I think it's safe to say that this is fixed now. I will close this issue now. But if for some reason I encounter the issue again, I might respond on this ticket again.

For now, all is good. Thanks for observing this issue with me.
